nav2_controller/src/nav2_controller.cpp

```diff
@@ -22,8 +22,9 @@
 #include "nav_2d_utils/conversions.hpp"
 #include "nav2_util/node_utils.hpp"
 #include "nav2_util/geometry_utils.hpp"
-#include "nav2_controller/progress_checker.hpp"
 #include "nav2_controller/nav2_controller.hpp"
+#include "nav2_controller/plugins/simple_progress_checker.hpp"
```

Comment:
If you're using pluginlib, you shouldn't need these headers

Reply:
Removed.